templates/CRM/Contribute/Form/Contribution/Confirm.tpl

```diff
@@ -153,7 +153,7 @@
 
   {if $onbehalfProfile && $onbehalfProfile|@count}
     <div class="crm-group onBehalf_display-group label-left crm-profile-view">
-      {include file="CRM/UF/Form/Block.tpl" fields=$onbehalfProfile prefix='onbehalf'}
+      {include file="CRM/UF/Form/Block.tpl" fields=$onbehalfProfile prefix='onbehalf'hideFieldset=false}
```

Comment:
There's a space missing before hidefieldset. I almost missed seeing the error since even with this PR there's still a ton of notices!

Reply:
thanks @demeritcowboy - fixed

Yeah - I've really focussed my cleanup on the event side of things - so those pages look much better than the online contribution page

Reply:
Ok I'm almost reviewed about as much as I'm going to, just trying to see why the on-behalf-of title is displaying twice, which seems to be from this PR somewhere, and then there's also a js syntax error which is probably from something else.